src/characterDetails.js

```diff
@@ -31,14 +31,28 @@ class CharacterDetails extends Component {
             bad: this.state.bad, 
             species_id: this.getSpeciesId() 
         };
-        console.log(newCharacterData)
         await updateCharacter(newCharacterData);
     }
 
+    handleDeleteEvent = async (e) => {
+        e.preventDefault()
+        console.log(this.state.id)
+        const newCharacterData = {
+            id: this.state.id,
+            name: this.state.name,
+            bad: this.state.bad, 
+            species_id: this.getSpeciesId() 
+        };
+        await deleteCharacter(newCharacterData);
```

Comment:
Great job putting together the delete function! Since you're only deleting the object, you don't actually need to pass the object through in the `body` and instead can simply pass the `id` in the URL params. Once SQL sees the `id`, it will delete the entry with that associated `id`.